### PR TITLE
Update reader sqlrace

### PR DESCRIPTION
**Pull request by owentmfoo**

### CODE REVIEW

File: Python/stream_reader_sqlrace/atlas_session_writer.py [hunk 1]
```diff
@@ -234,26 +317,22 @@ def add_data(
         databytes = bytearray(len(data) * 8)
         for i, value in enumerate(data):
             new_bytes = struct.pack("d", value)
-            databytes[i * 8 : i * 8 + len(new_bytes)] = new_bytes
```
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/atlas_session_writer.py]: This is just whitespace change. Probably remove it

File: Python/stream_reader_sqlrace/main.py [hunk 2]
```diff
@@ -162,19 +361,102 @@ async def handle_periodic_packet(self, packet: open_data_pb2.PeriodicDataPacket)
...
+                self.session_writer.add_details(key, value.value)
+            elif any_value.Is(wrappers_pb2.DoubleValue.DESCRIPTOR):
+                value = wrappers_pb2.DoubleValue()
+                any_value.Unpack(value)
+                self.session_writer.add_details(key, value.value)
+            else:
+                print(f"Unsupported value type for metadata: {key}")
+
+    async def handle_event_packet(self, packet: open_data_pb2.EventPacket):
+        # TODO: deal with missing configs
```
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/main.py]: Looks like you are already doing dealing with missing config. You should remove this TODO

File: Python/stream_reader_sqlrace/main.py
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/main.py:1]: This main is absolutely massive. I'd say try to break it down more.

File: Python/stream_reader_sqlrace/atlas_session_writer.py
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/atlas_session_writer.py:1]: This file is absolutely massive. Try to find a way to break it down.

File: Python/stream_reader_sqlrace/additional_requirements.txt [hunk 3]
```diff
@@ -1 +1,2 @@
-pythonnet==3.0.1
```
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/additional_requirements.txt]: Why is pythonnet being removed and readded here?

File: Python/stream_reader_sqlrace/main.py (hunk 2 above)
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/main.py]: removed todo

File: Python/stream_reader_sqlrace/atlas_session_writer.py (hunk 1 above)
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/atlas_session_writer.py]: reformatted to be in line with PEP8

File: Python/stream_reader_sqlrace/additional_requirements.txt (hunk 3 above)
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/additional_requirements.txt]: Extra line return character added, hence the diff

**Review by salman-mclarenapplied — APPROVED** [review]

**Pull request merged**